docs/user-guide/pods.md

```diff
@@ -50,42 +50,92 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: GENERATED_TOC -->
 
-In Kubernetes, rather than individual application containers, _pods_ are the smallest deployable units that can be created, scheduled, and managed.
+_pods_ are the smallest deployable units that can be created and managed in
+Kubernetes.
 
 ## What is a _pod_?
 
-A _pod_ (as in a pod of whales or pea pod) corresponds to a colocated group of applications running with a shared context. Within that context, the applications may also have individual cgroup isolations applied. A pod models an application-specific "logical host" in a containerized environment. It may contain one or more applications which are relatively tightly coupled &mdash; in a pre-container world, they would have executed on the same physical or virtual host.
+A _pod_ (as in a pod of whales or pea pod) is a group of applications, such as
```

Comment:
A pod encompasses one container (and sometimes more than one), the storage volumes that container uses, and options about how to run the container.